TicTacToe2/TicTacToe/Player.cs

```diff
@@ -6,5 +6,18 @@ namespace TicTacToe2.TicTacToe
 {
     class Player
```

Comment:
A few things about this player class:

In a game of tic tac toe there are two players. Each player has a symbol (X or O). Players are responsible only for deciding where to place their mark. The interface (+ constructor) for a player would look like

```c#
public Player(char symbol)
{
}

public void GetMove(out int x, out int y) // we need to return multiple values
{
}

public char GetSymbol()
{
}
```

C# has a feature called properties (https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/properties) which would be more appropriate for handling symbol in the interface

```c#
public char Symbol { get; }

public Player(char symbol)
{
}

public void GetMove(out int x, out int y) // we need to return multiple values
{
}
```